core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -21,10 +21,10 @@ <h3 class="activity-title"><[getCollectionTitle()]></h3>
           </li>
 
           <li>
-            <span ng-if="!wasRecentlyUpdated()">
+            <span ng-if="!wasRecentlyUpdated() || inCreatorDashboard">
               <[getLastUpdatedDatetime()]>
             </span>
-            <span ng-if="wasRecentlyUpdated()" class="recently-updated">
+            <span ng-if="wasRecentlyUpdated() && !inCreatorDashboard" class="recently-updated">
```

Comment:
I also never really saw the use of having "UPDATED". So I think @jblair87 is right and I can get rid of it.